### ISSUE TITLE
Disable autoselect instances

### ISSUE THREAD

**Issue by iotch** [zalmoxisus/redux-devtools-extension#303]
Hello. Is there a way to disable autoselect instances or have display settings remembered per browser tab?
Now, if I select to display, say, chart and desired instance and then hit F5 my selection won't be remembered. 
Thanks!

**Comment by zalmoxisus** [comment]
By `F5` you mean reloading the app's page or the extension window? When you reload the page, the instance is destroyed and another one is created. If you persist the state (click the `Persist` button from the bottom), the instance will not be destroyed.

If you use the extension from Chrome devtools panel it will be sticked to the current tab's instance by default (instead of autoselecting), so it will do the trick (unless you have multiple stores).

**Comment by iotch** [comment]
I'm using extension from Chrome devtools panel. It does not preserve selected panes between html page reloads in chrome 47. I just installed chrome 56 and it works fine. However, not in chrome 47.
Anyway, if chrome 47 is not supported, this is apparently not a bug.

**Comment by zalmoxisus** [comment]
If tab id is not detected (to select the tab's instance by default), then could be that `chrome.devtools.inspectedWindow.tabId` wasn't worked as expected in chrome 47. I guess you cannot downgrade Chrome to debug that.

**Comment by iotch** [comment]
Nope, sorry, disregard my previous comment. Same behavior with Crhome 56. As far as I understand, to keep redux-devtools-extension to open the same pane (log monitor/inspector/chart) after page reload, you need to select the desired pane, close and reopen chrome devtools to save changes. Changes are not get stored on pane selection, only on close. Am i right?

**Comment by zalmoxisus** [comment]
Ah, I thought the instance is not remembered. But I see the selected monitor is not persisted for devpanel.

**Comment by iotch** [comment]
Yes, and action/state/diff/test buttons, and three/chart/raw tabs are not persisted as well. And in this case reopen doesn't help.

**Comment by zalmoxisus** [comment]
Persisting the selected monitor was broken before, but the other part is not implemented yet. I think I'll better add it to `3.0` milestone, since we're changing it completely. Thanks for the report!
